src/components/sections/Newsletter/Newsletter.tsx

```diff
@@ -0,0 +1,82 @@
+import type { ComponentPropsWithRef, FormEvent, ReactNode } from 'react'
+import React, { forwardRef, useRef } from 'react'
+import { Form, Label, Input, Button } from '@faststore/ui'
+
+export interface NewsletterProps
+  extends Omit<ComponentPropsWithRef<'form'>, 'onSubmit'> {
+  /**
+   * Heading for the section.
+   */
+  heading: ReactNode
```

Comment:
The reason why you are calling it `heading` is because the user can pass the tag (h2, h3, h4, h5 ...), right?
Because until now we are using `title` on the other components. 🤔

Reply:
I [renamed](https://github.com/vtex-sites/base.store/pull/242/commits/8b9a21175198837095ed3dffcfba79e121f33aec) this prop to `title`. Also [renamed](https://github.com/vtex-sites/base.store/pull/242/commits/60dab95a2f25873d0b0a4fc813db4d2ee6e3c265) `description` to `subtitle` sincei it comes right below the title.